mesonbuild/interpreter/type_checking.py

```diff
@@ -294,6 +294,7 @@ def _env_convertor(value: _FullEnvInitValueType) -> EnvironmentVariables:
 )
 
 def _override_options_convertor(raw: T.Union[str, T.List[str], T.Dict[str, ElementaryOptionValues]]) -> T.Dict[OptionKey, ElementaryOptionValues]:
```

Comment:
Dead function. I haven't followed the code enough, but I wonder how much the tests cover all the cases... This is the single thing that I'm most nervous about.